build/build.rb

```diff
@@ -89,6 +89,9 @@ def verify_run(output)
     file_name = "#{name}-#{target}-#{build_label}-#{timestamp}-#{git_checkout.commit_sha}.#{FILE_TYPE[imgfac_target]}"
     destination = DESTINATION_DIRECTORY.join(file_name)
     $log.info `mv  #{source} #{destination}`
+    $log.info "Generating image checksums"
+    $log.info `cd #{DESTINATION_DIRECTORY}/; /usr/bin/md5sum * >> md5sum`
+    $log.info `cd #{DESTINATION_DIRECTORY}/; /usr/bin/sha256sum * >> sha256sum`
```

Comment:
Won't this end up creating a checksum on the md5sum file?  Also, I'm concerned about the `cd`.  If someone were to write more code after this, they may not realize that you changed directories.  It might be better to use `Dir.chdir` with the block notation around these two calls, or perhaps just use the dest direcotry in the calls themselves...something like:

```ruby
$log.info `/usr/bin/md5sum #{DESTINATION_DIRECTORY.join('*')} >> #{DESTINATION_DIRECTORY.join('md5sum')}`
```

But that still doesn't solve the sha256sum of the md5sum file.  Maybe write to temp dir and move?